SpringProject/src/main/java/com/gradproject2019/auth/persistance/Token.java

```diff
@@ -0,0 +1,54 @@
+package com.gradproject2019.auth.persistance;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tokens")
+public class Token {
+    @Id
+    private Long userId;
+    private String token;
```

Comment:
Just wondering, is this actually a String or is it a UUID?

Reply:
Hi Zabi, it should be a UUID so we've changed the type from String to UUID. Thanks!!